app/directives/proposals/proposals.directive.js

```diff
@@ -87,18 +91,30 @@ angular.module('dashboard')
 
             function deleteProposal(proposal) {
                 if (proposal instanceof Object) {
-                    AhjoProposalsSrv.delete(proposal).$promise.then(function(response) {
-                        $log.debug("dbProposals: delete then: " + JSON.stringify(response));
-                    }, function(error) {
-                        $log.error("dbProposals: delete error: " + JSON.stringify(error));
-                    }, function(notify) {
-                        $log.debug("dbProposals: delete notify: " + JSON.stringify(notify));
-                    }).finally(function() {
-                        $log.debug("dbProposals: delete finally: ");
-                    });
-                }
-                else {
-                    $log.error('dbProposals: deleteProposal parameter invalid');
+                    if (proposal.isNew && self.proposals instanceof Array) {
+                        for (var index = 0; index < self.proposals.length; index++) {
+                            if (angular.equals(proposal, self.proposals[index])) {
+                                self.proposals.splice(index, 1);
+                                continue;
```

Comment:
I guess `continue` has no effect in this case because deleteProposal's `for` doesn't have anything to be executed for an iteration after the above row. Loop should jump to next `for` loop iteration in any case even without the continue.
The `continue` would have effect if there was some code inside the `for` after the `if`.

reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue

Btw in my opinion  `continue` adds just an extra amount of maintenance effort for the code later on, just like `goto`. Usually there's always just as simple and efficient means to do the same thing without `continue`.